daft/io/_json.py

```diff
@@ -56,5 +56,5 @@ def read_json(
         storage_config = StorageConfig.native(NativeStorageConfig(True, io_config))
     else:
         storage_config = StorageConfig.python(PythonStorageConfig(io_config=io_config))
-    builder = get_tabular_files_scan(path, schema_hints, file_format_config, storage_config=storage_config)
+    builder = get_tabular_files_scan(path, schema_hints, None, file_format_config, storage_config=storage_config)
```

Comment:
Maybe we should also do a quick follow-up here to enable override next for JSON, to be consistent

Reply:
Agreed